wdl/tasks/sed.wdl

```diff
@@ -0,0 +1,30 @@
+version 1.0
+
+
+import "../structs/resources.wdl"
+
+
+task remove_whitespace_from_end_of_lines{
```

Comment:
You don't have to change this but maybe `remove_trailing_whitespaces`?

Reply:
I like that naming better than the original. Just being a clumsy foreigner here..